File: Part4/Fortran/Function_IncorrectCapitalization.f90
program error2_wrong_capitalization
    implicit none
    character( len=100 ) :: user_input
    integer :: entered_integer, ios
    
    WRITE( *, * ) 'Type 1 and press enter.'
    READ( *, '(A)' ) user_input
    
    ! Using inconsistent case (this actually works in Fortran but shows style issue)
    entered_integer = INT_CONVERT( user_input )
    
    write( *, * ) 'The user entered the integer ', entered_integer
    
end program error2_wrong_capitalization

File: Part4/Rust/Function_IncorrectCapitalization.rs
use std::io;

fn main()
{
    let mut user_input = String::new();
    
    println!( "Type 1 and press enter." );
    io::stdin().read_line( &mut user_input ).expect( "Failed to read line" );
    
    // Function name has wrong capitalization
    let entered_integer: i32 = user_input.trim().Parse::<i32>().unwrap();
    
    println!( "The user entered the integer {}", entered_integer );
}
